src/controllers/metadata.js

```diff
@@ -1,21 +1,22 @@
 'use strict'
 const config = require('../services/config')
 
+const base = config.getIn(['server', 'base_uri'])
 const metadata = {
   currency_code: config.getIn(['currency', 'code']),
   currency_symbol: config.getIn(['currency', 'symbol']),
   condition_sign_public_key: config.getIn(['keys', 'ed25519', 'public']),
   notification_sign_public_key: config.getIn(['keys', 'notification_sign', 'public']),
   urls: {
-    health: '/health',
-    transfer: '/transfers/:id',
-    transfer_fulfillment: '/transfers/:id/fulfillment',
-    transfer_state: '/transfers/:id/state',
-    connectors: '/connectors',
-    accounts: '/accounts',
-    account: '/accounts/:name',
-    subscription: '/subscriptions/:id',
-    subscription_notification: '/subscriptions/:subscription_id/notifications/:notification_id'
+    health: base + '/health',
```

Comment:
Just checked the README (https://github.com/interledger/five-bells-connector/#general)


> `CONNECTOR_HOSTNAME` (default: [your hostname]) Publicly visible hostname. This is important for things like generating globally unique IDs. Make sure this is a hostname that all your clients will be able to see. The default should be fine for local testing.